src/analyzing/AST_to_IL.ml

```diff
@@ -1108,6 +1108,17 @@ and stmt_expr env ?e_gen st =
   | G.Return (t, eorig, _) ->
```

Comment:
Not sure what would be a good changelog entry here but we should add something.

Reply:
Added changelog.